### ISSUE TITLE
An "ELI5" document for turning a RasPi into an ImmichFrame

### ISSUE THREAD

**Issue by voyager529** [immichFrame/ImmichFrame#235]
### Description

Hi friends!

So, I'm trying to get this to work, and it *seems* simple, but I submit that there's room for some clarity.

The first thing I learned in installing this on a Raspberry Pi is that it requires the FULL, kitchen-sink edition of RaspberryPi OS. I spent last night trying to get LibGTK+ installed on DietPi, but Synaptic seemed to just like complaining about the absence of dependencies instead of just installing them...same thing on RasPiOS Lite. Installing the full GUI was the ticket; that was the *real* dependency for all practical purposes.

From there, I did a ```dpkg -i immichframe_0.1.0_LINUX_arm64.deb```, and it finally installed. Huzzah!

The documentation then says to ```cd``` into the folder with ImmichFrame downloaded, but since I did the Debian installer (the only option I saw listed for ARM64), that doesn't exactly seem to apply...so I just ran ```DISPLAY=:0.0 ./Immich_Frame``` via ssh, which does invoke the application, but then I get told ```Gtk-WARNING **: 09:43:16.039: cannot open display: :0.0```

I tried a few different numbers for displays (I have one of the RasPis with multiple HDMI outputs), no change in behavior.

Meanwhille, config variables are listed, but it's super unclear where to put them...do I add them to a .env file somewhere? is there a .conf file that ImmichFrame can read from on startup?

Thanks for looking into this for me =)

### Use Case

I'd like to think that many users would benefit from some of these getting-started questions being enumerated.

### Pre-Submission Checklist

- [X] This is a feature request and not a bug report

- [X] I have checked for related issues and checked the documentation

**Comment by 3rob3** [comment]
I agree with you the documentation for raspberry pi is not great, but it is very difficult with the different OS's, and even OS version (not to mention Wayland/Wayfire/X11/etc). I find using the chromium browser works best on Pi 4 and 5's. I have a similar reply here with some more details:
https://github.com/immichFrame/ImmichFrame/issues/233

Also, anyone is welcome to contribute to the documentation. If you get it configured good and want to share we would love the contribution!
